Formula/aicommits.rb

```diff
@@ -0,0 +1,32 @@
+require "language/node"
+
+class Aicommits < Formula
+  desc "Writes your git commit messages for you with AI"
+  homepage "https://github.com/Nutlope/aicommits"
+  url "https://registry.npmjs.org/aicommits/-/aicommits-1.5.1.tgz"
+  sha256 "cc2d9011a3ef1eb7317078c9154753412f3cb42f8a73dccc1cd8f84744c57a72"
+  license "MIT"
+
+  livecheck do
+    url :stable
+  end
+
```

Comment:
```suggestion
```

Reply:
Hmm, I though I already fixed this with my force push 🤔